.github/workflows/tests.yml

```diff
@@ -204,6 +204,37 @@ jobs:
             test-report.xml
           retention-days: 1
 
+  # linux benchmarks
+  linux-benchmarks:
+    # only run test suite if there are code changes
+    needs: changes
+    if: needs.changes.outputs.code == 'true'
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.11', '3.12']
```

Comment:
Not sure if it makes sense to benchmark different versions of Python here, and if it should be 3.8 and 3.12 TBH. Any thoughts on this?

Reply:
I would just define a single Python version for now. I don't think it will bring much to run benchmarks for two different versions because we're mostly focused on a relative measurements between conda versions or commits and not between Python versions (although that may be interesting to know).

If we do add different versions, it makes sense to add the latest and oldest supported versions (3.8 and 3.12).

Reply:
Let's make it simpler and just use one version